roles/openshift_node40/tasks/config.yml

```diff
@@ -62,13 +62,21 @@
       podman_flags: "--privileged --rm -ti {{ release_image_mcd.stdout }}"
       podman_mounts: "-v /:/rootfs -v /var/run/dbus:/var/run/dbus -v /run/systemd:/run/systemd"
       mcd_command: "start --node-name {{ ansible_hostname }} --once-from {{ ign_file }}"
-    # MCD reboots the machine
-    ignore_unreachable: true
-  # Wait for the host to come back and reset 'unreachable' status
+    # MCD reboots the machine, run the task but do not wait for completion
+    register: manifest_apply
+    async: 900  # 15 minutes
+    poll: 0
+
+  # Wait for the host to come back
   - wait_for_connection: {}
-  # Clear unreachable status
-  - name: clear any host unreachable error messages.
-    meta: clear_host_errors
+
+  # If the job fails, the async job status will find rc != 1 and will fail here
+  # When the job is successful, Ansible does not update this job status due to
+  # the host rebooting
+  - name: Check manifest apply status
+    async_status:
+      jid: "{{ manifest_apply.ansible_job_id }}"
+
```

Comment:
I guess it needs another `wait_for_connection`. First could have occurred when MCO is laying out files but not rebooted yet - so first `wait_for_connection` would pass. Then `async_status` would wait for command to be cancelled (during the reboot). Now another `wait_for_connection` might be required